src/main/steps/applicant1/their-email-address/content.ts

```diff
@@ -54,17 +55,21 @@ const cy = ({ partner }) => ({
       incorrect:
         'Rydych wedi rhoi cyfeiriad e-bost ac wedi nodi nad ydych yn gwybod beth yw ei gyfeiriad/chyfeiriad e-bost. Dim ond un y gallwch ei wneud cyn parhau.',
       invalid: 'Rydych wedi rhoi cyfeiriad e-bost annilys. Gwiriwch ef a nodwch ef eto cyn parhau.',
+      sameEmail: `You have entered your own email address. You need to enter your ${partner}'s email address before continuing.`, //todo NFDIV-2467
     },
   },
 });
 
 export const form: FormContent = {
-  fields: {
+  fields: userCase => ({
     applicant2EmailAddress: {
       type: 'text',
       label: l => l.applicant2EmailAddress,
       labelSize: null,
       validator: (value, formData) => {
+        if (value === userCase.applicant1Email) {
+          return 'sameEmail';
+        }
```

Comment:
better to put this into src/main/app/form/validation.ts as we can then test it too.